content/en/getting-started/installation.md

```diff
@@ -58,10 +61,27 @@ If you have problems with permissions in MacOS X Sierra, install with `python3 -
 
 {{< tab header="Linux" >}}
 <p>
-You can just download the respective binary for your architecture directly:<br>
+You can download the respective binary for your architecture directly:<br>
 {{< cli-binary-download os="linux" >}}
 </p>
 <p>
+or use this <code>curl</code> command:
+</p>
+<p>
+For <i>amd64</i>:
+{{< command >}}
+$ curl -Lo ~/Downloads/localstack-cli-{{< localstack-latest-version >}}-linux-amd64-onefile.tar.gz \
```

Comment:
nit: I think we should not assume that there is a `Downloads` folder in the home directory of the user, but just download it into the current working directory.

Reply:
I agree, I was making the curl command consistent with the tar unpack instructions. I'll update both to use the CWD :+1: